src/datanode/src/config.rs

```diff
@@ -305,6 +305,8 @@ pub struct DatanodeOptions {
     pub meta_client: Option<MetaClientOptions>,
     pub wal: DatanodeWalConfig,
     pub storage: StorageConfig,
+    pub parallelism: Option<usize>,
+    pub semaphore_timeout_ms: u64,
```

Comment:
I thikn semaphore is an implementation related concept, so we better not to expose this name in configuration and error output. 

Clickhouse has a configuration item called `max_concurrent_queries` can be a good name for `parallelism`.

Something like `concurrent_query_limiter_timeout` can be good, but you might have better names for this.

Reply:
thanks for the suggestion I have updated the naming and configurations docs